city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,129 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiNorthernIlUniversitySpider(CityScrapersSpider):
+    name = "chi_northern_il_university"
+    agency = "Northeastern Illinois University"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.neiu.edu/about/board-of-trustees/board-meeting-materials"]
+
+    def parse(self, response):
+        self.link_date_map = defaultdict(list)
+        link_list = response.xpath(
+            '//div[@class="field field--name-field-generic-body field--type-text-long \
+field--label-hidden field--item"]/h3'
+        )
+        for title in link_list:
+            full_str = title.xpath(".//text()").extract()[0]
+            date = full_str.split("-")[0].split("(")[0]
+
+            date_obj = datetime.strptime(date.strip(" "), '%B %d, %Y')
+            new_date_obj = datetime(date_obj.year, date_obj.month, date_obj.day, 13, 00)
+            for one_day_links in title.xpath('.//following-sibling::ul[1]/li'):
+                href = one_day_links.xpath('.//@href').get()
+                name = one_day_links.xpath('.//text()').extract()
+                if href is None:
+                    link = ''
+                else:
+                    link = response.urljoin(href)
+                self.link_date_map.setdefault(new_date_obj, []).append({
+                    "title": ''.join(name),
+                    "href": link
+                })
+        yield response.follow(
+            "https://www.neiu.edu/about/board-of-trustees/calendar-of-meetings",
+            callback=self._parse_detail
+        )
+
+    def _parse_detail(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        items_list = response.xpath(
+            '//div[@class="field field--name-field-generic-body field--type-text-long \
+field--label-hidden field--item"]'
+        )
+        year_list = items_list.xpath('.//h2')
+        for years in year_list.extract():
+            year = re.findall(r'\d{4}', years)[0]
+            items = year_list.xpath('.//following-sibling::ul[1]/li').extract()
+            for item in items:
```

Comment:
We might want to create an `item_str` variable in here to reduce the amount of rework we're doing in parsing out the text. Also, we can do it a little more simply with text selectors and regex

```python
# Combines all text nodes into one string and replaces duplicate spaces
item_str = re.sub(r"\s+", " ", " ".join(item.css("*::text").extract())).strip()
```